apps/server/lib/lexical/server/code_intelligence/completion/env.ex

```diff
@@ -291,6 +312,19 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Env do
     end
   end
 
+  @spec prefix_aliases(t()) :: [atom() | String.t()]
+  def prefix_aliases(env) do
```

Comment:
what does this do? Functions that deal with AST arcana need to be documented. I would also like to see documentation around the different clauses in the `find_value` function, telling future us what's being found.